resharper/test/data/CSharp/Resolve/UnityObjectTypeInStringLiteral/CodeCompletion/AddComponent01.cs

```diff
@@ -0,0 +1,9 @@
+using UnityEngine;
+
+public class Test
+{
+    public void Method(GameObject go)
+    {
+        go.AddComponent("{caret}");
+    }
+}
```

Comment:
```
namespace A
{
	public class Test2 : MonoBehaviour
	{
				
	}
}

public class Test : MonoBehaviour
{
	void Update ()
	{
			Debug.Log(GetComponent("Tes{caret}")); 
	}
}
```
Code completion do not suggest `Test2`

Reply:
No, it can't. As mentioned in the PR details above. We can't provide code completion for ALL types, the list is too big and too full of items that are irrelevant.